dart-sample/module1/src/main/java/com/f2prateek/dart/model/NavigationActivity.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013 Jake Wharton
+ * Copyright 2014 Prateek Srivastava (@f2prateek)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.f2prateek.dart.model;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.widget.TextView;
+import butterknife.ButterKnife;
+import butterknife.BindView;
+import dart.Dart;
+
+public class NavigationActivity extends Activity {
```

Comment:
Should we change the activity name?

I find it confusing to use NavigationActivity as it is not related to the navigation layer.

As it belongs to another module and that's the point of the example, we could rename it to Model1Activity. Same for the service.

And the sampleactivity could be renamed to SameModuleActivity.

Do you mind to rename the NavigationModels using the following pattern: `FooActivityNavigationModel`?

Thx

Reply:
Sure, I thought it to rename to Module1Activity initially. WDYT?